### PR TITLE
the manifest for meta-xt-ihor-usyk layer is created

### PR DESCRIPTION
**Pull request by iusyk**

### CODE REVIEW

File: prod_ihor_usyk/domd.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    <default remote="github" sync-j="10" sync-c="true" />
+
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="thud-dev" revision="2a535519e1808bb3a43e88a6c5dcc0556fc7101d" />
+    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
+    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="thud" revision="fb6192aa2c5df8e80c5e6d4fa5448d574332f68f" />
+    <project remote="epam" name="epmd-aepr/meta-aos" path="meta-aos" upstream="thud" revision="thud" />
```
Comment (iartemenko) [inline prod_ihor_usyk/domd.xml:20]: You might want to add META compiler for PVR UM/KM, please see prod-devel-src manifest file.

File: prod_ihor_usyk/domu.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    <default remote="github" sync-j="10" sync-c="true" />
+
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
+    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
+    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />
+</manifest>
```
Comment (iartemenko) [inline prod_ihor_usyk/domu.xml:18]: Ditto

**Review by iartemenko — CHANGES_REQUESTED** [review]

File: prod_ihor_usyk/dom0.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
+    <remote name="openembedded" fetch="git://git.openembedded.org" />
+    <remote name="linaro" fetch="git://git.linaro.org" />
+
+    <default remote="github" sync-j="10" sync-c="true" />
+
+    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
```
Comment (iartemenko) [inline prod_ihor_usyk/dom0.xml]: Revision is incorrect see prod-devel manifest file.

File: prod_ihor_usyk/domd.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <remote name="linaro" fetch="git://git.linaro.org" />
+    <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
+
+    <default remote="github" sync-j="10" sync-c="true" />
+    <project remote="epam" name="epmd-aepr/img-proprietary" path="proprietary" upstream="master" revision="master" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="thud-dev" revision="2a535519e1808bb3a43e88a6c5dcc0556fc7101d" />
+    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
```
Comment (iartemenko) [inline prod_ihor_usyk/domd.xml]: Ditto

File: prod_ihor_usyk/domd.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+<manifest>
+    <remote name="github" fetch="git://github.com" />
+    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
+    <remote name="openembedded" fetch="git://git.openembedded.org" />
+    <remote name="linaro" fetch="git://git.linaro.org" />
+    <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
+
+    <default remote="github" sync-j="10" sync-c="true" />
+    <project remote="epam" name="epmd-aepr/img-proprietary" path="proprietary" upstream="master" revision="master" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="thud-dev" revision="2a535519e1808bb3a43e88a6c5dcc0556fc7101d" />
```
Comment (iartemenko) [inline prod_ihor_usyk/domd.xml]: Revision is incorrect see prod-devel manifest file.

File: prod_ihor_usyk/domu.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+    <default remote="github" sync-j="10" sync-c="true" />
+
+    <project remote="epam" name="epmd-aepr/img-proprietary" path="proprietary" upstream="master" revision="master" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
+    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
+    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />
```
Comment (iartemenko) [inline prod_ihor_usyk/domu.xml]: Revision is incorrect see prod-devel manifest file.

Comment (iartemenko) [reply prod_ihor_usyk/domu.xml]: Whole manifest file is incorrect.

File: prod_ihor_usyk/dom0.xml (hunk 3 above)
Comment (iusyk) [reply prod_ihor_usyk/dom0.xml]: please, find any differences - below the content of prod_devel/dom0.xml
<?xml version="1.0" encoding="UTF-8"?>
<manifest>
    <remote name="github" fetch="git://github.com" />
    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
    <remote name="openembedded" fetch="git://git.openembedded.org" />
    <remote name="linaro" fetch="git://git.linaro.org" />

    <default remote="github" sync-j="10" sync-c="true" />

    <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
    <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
</manifest>

File: prod_ihor_usyk/domu.xml (hunk 6 above)
Comment (iusyk) [reply prod_ihor_usyk/domu.xml]: there is just a difference - this domu has img-proprietary
that is it 
below the body form prod-devel
<remote name="github" fetch="git://github.com" />
    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
    <remote name="openembedded" fetch="git://git.openembedded.org" />
    <remote name="linaro" fetch="git://git.linaro.org" />

    <default remote="github" sync-j="10" sync-c="true" />

    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />

File: prod_ihor_usyk/dom0.xml (hunk 3 above)
Comment (iartemenko) [reply prod_ihor_usyk/dom0.xml]: https://github.com/xen-troops/manifests/blob/master/prod_devel/dom0.xml
Ok, let's see it together...

**Comment by iusyk** [comment]
it was a test

**Pull request closed without merging**